### PR TITLE
Build hdc-algo meta package

### PR DESCRIPTION
**Pull request by Kirill888**
- hdc-algo=X.X.X depends on hdc-algo-core=X.X.X + numba-scipy

### CODE REVIEW

**Comment by Kirill888** [comment]
produced packages example

https://github.com/WFP-VAM/hdc-algo/suites/16892741108/artifacts/964968372

Once merged use manual workflow run here to publish specific branch or ideally tag to `wfp-ram` channel:

https://github.com/WFP-VAM/hdc-algo/actions/workflows/conda.yaml

image

File: conda/recipe-meta/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - numba-scipy >=0.3.1
+    - scipy
+
+test:
+  imports:
+    - hdc.algo
+
+about:
+  home: https://github.com/WFP-VAM/hdc-algo/
+  summary: xarray tools for seasonal monitor
```
Comment (valpesendorfer) [inline conda/recipe-meta/meta.yaml:33]:
```suggestion
  summary: algorithmic components and xarray tools used in the Humanitarian Data Cube
```

**Comment by valpesendorfer** [comment]
> Once merged use manual workflow run here to publish specific branch or ideally tag to `wfp-ram` channel:

with "tag `wfp-ram` channel" you mean?

How should we go forward w/ publishing new versions, should we start an actual versioning or continue with dev-releases? If so do we manually need to set the number (i.e. check which one is current and increment)?

**Comment by Kirill888** [comment]
Ideally we would implement github workflow that triggers when tag is created, runs all the tests, creates all the conda and wheel packages, and if all good uploads those artifacts.

But for manual process current steps are:

1. Bump version and commit this to main
2. Make sure tests pass
3. Create a tag
4. Manually run conda workflow on a tag and tick "upload checkbox"

OR dirty case:
- just run conda workflow from whatever branch overriding version from github action dialog

**Comment by Kirill888** [comment]
> with "tag `wfp-ram` channel" you mean?

sorry, I meant build conda package from a specific branch, or better yet, create a tag and run conda workflow on that tagged commit.

**Comment by valpesendorfer** [comment]
ok understood. So after merge here we could bump version in separate PR ( I think we should go to `0.2` given changes in modules) and then create a tag and run workflow

**Review by valpesendorfer — APPROVED** [review]

**Pull request merged**